tests/lean/run/repr_empty.lean

```diff
@@ -0,0 +1,27 @@
+/-!
+# Validation of `Repr Empty` instance
+
+While it may seem unnecessary to have Repr, it prevents the automatic derivation
+of Repr for other types when `Empty` is used as a type parameter.
+
+This is a simplified version of an example from the "Functional Programming in Lean" book.
+The Empty type is used to exlude the `other` constructor from the `Prim` type.
+-/
+
+inductive Prim (special : Type) where
+  | plus
+  | minus
+  | other : special → Prim special
+deriving Repr
+
+-- Check that both Repr's work
```

Comment:
I'm not sure this is written down anywhere, but my understanding of the style is to use module docstrings for explanations of tests. Given that you already wrote about it in the main module docstring, this seems superfluous, but in any case, here's the syntax.
```suggestion
/-!
Check that both Repr's work
-/
```

Reply:
You're right, it's documented here which is how I knew to write the initial docstring: [Writing good tests](https://github.com/leanprover/lean4/blob/master/doc/dev/testing.md#writing-good-tests).  The second comment was a mistake - I'll be aware in the future.